feature_tests/js2/api/Foo.mjs

```diff
@@ -8,12 +7,9 @@ import wasm from "./diplomat-wasm.mjs";
 import * as diplomatRuntime from "./diplomat-runtime.mjs";
 
 
-
-
 const Foo_box_destroy_registry = new FinalizationRegistry((ptr) => {
     wasm.Foo_destroy(ptr);
```

Comment:
Question: does this also need to destroy `aEdge` instances? I guess not; they are tracked on their own.

Reply:
#525